merkato/merkato.py

```diff
@@ -2,24 +2,26 @@
 import json
 
 from merkato.exchanges.tux_exchange.exchange import TuxExchange
-from merkato.utils import create_price_data
-from merkato.constants import BUY, SELL, ID, PRICE, LAST_ORDER, ASK_RESERVE, BID_RESERVE, ASK_BUDGET, BID_BUDGET
-from merkato.utils.database_utils import update_merkato, get_all_merkatos_from_exchange
+from merkato.constants import BUY, SELL, ID, PRICE, LAST_ORDER, ASK_RESERVE, BID_RESERVE, EXCHANGE
+from merkato.utils.database_utils import update_merkato, insert_merkato
+from merkato.exchanges.tux_exchange.utils import translate_ticker
+from merkato.utils import create_price_data, validate_merkato_initialization, get_relevant_exchange
 from math import floor
 
 DEBUG = True
 
 
 class Merkato(object):
-    def __init__(self, configuration, exchange, coin, base, spread, ask_budget, bid_budget):
```

Comment:
budgets aren't needed to instantiate? that will change the GUI, which is fine, but how do you specify how much to spend on each merkato?

Reply:
After speaking yesterday, we decided only reserve and the amount on the books are necessary for operation, 'budget' can be queried by updating a merkato and adding together the reserve + orders on the books.

Reply:
Basically it was a duplicate field

Reply:
so how do i start a new one and say only use 30% of my idle BTC?

Reply:
@livinginformation my point is, I can't currently launch a merkato and watch it work to see if TestExchange is good to go.